.binder/environment.yml

```diff
@@ -59,6 +60,8 @@ dependencies:
   - jupyterlab-geojson >=3,<4
   - jupyterlab-kernelspy
   - jupyterlab_miami_nights
+  - networkx
+  - pandas
```

Comment:
We should be able to remove these two from the `environment.yml` files. They are only needed in the kernel.

We still need to keep `ipycytoscape` to pick up the JupyterLab prebuilt extension (using the new system in 3.0). There is some info in the docs for reference: https://jupyterlite.readthedocs.io/en/latest/configuring.html#adding-extensions

Reply:
At some point later we could also expand the contributing docs to show how to add new libraries and example notebooks to the demo site: https://jupyterlite.readthedocs.io/en/latest/contributing.html